src/definitions/modules/dropdown.js

```diff
@@ -366,11 +365,22 @@ $.fn.dropdown = function(parameters) {
             }
             if( module.is.search() && !module.has.search() ) {
               module.verbose('Adding search input');
-              $search = $('<input />')
+              if($module.prev('label').length) {
+                $search = $('<input />')
                 .addClass(className.search)
                 .prop('autocomplete', module.is.chrome() ? 'fomantic-search' : 'off')
+                .attr('aria-labelledby',id+'_formLabel')
                 .insertBefore($text)
-              ;
+                ;
+                $module.prev('label').attr('id', id + '_formLabel');
```

Comment:
What if I already set `id` attribute to that label?

Reply:
thanks a lot for your objection, it is fixed